### ISSUE TITLE
Build ends up in bin/bin directory

### ISSUE THREAD

**Issue by stuaxo** [Shervanator/Engine#30]
I built this on Ubuntu and the output directory seems a bit odd 

I ran:
```
mkdir build
cd build
cmake ..
make
cd ..
```

The ended up with a directory ```bin/bin``` - it seems like it should just be ```bin```

**Comment by Shervanator** [comment]
yeah its because of this line: https://github.com/Shervanator/Engine/blob/master/CMakeLists.txt#L26 but if i remember correctly i had to do this to get on of the dependencies compiling properly :( i think it should be possible to clean up though..
